src/Configuration/ConfigDefinition.php

```diff
@@ -27,6 +27,7 @@ public function getParametersDefinition(): ArrayNodeDefinition
                 })
             ->end()
             ->children()
+                ->scalarNode('outputBucket')->isRequired()->cannotBeEmpty()->end()
```

Comment:
Takze uz nebude default bucket (asi sa to neda)? A UI to bude simulovat, generovat tento klu?

Reply:
ano, přesně tak... až doteď tomu tak bylo
https://github.com/keboola/google-analytics-extractor/blob/3.6.1/src/Keboola/GoogleAnalyticsExtractor/Configuration/ConfigDefinition.php#L22

Reply:
on vlastně ani nikdy default bucket v produkci nebyl... to co jsem posílal ^ je na starší verzi která je v produkci